### PR TITLE
added some basic translations

### PR DESCRIPTION
**Pull request by kriegerlukas**
Added some basic single word translations via google translate.

Please also check the file, i looks like there are some issues with the formatting. Sometimes the msgid is empty or there are multiple strings for a single translation msgid

### CODE REVIEW

File: locale/es_ES.UTF-8/LC_MESSAGES/palma.po [hunk 1]
```diff
@@ -254,7 +254,7 @@ msgstr ""
 
 #: index.php:1233
 msgid "Windows:"
-msgstr ""
+msgstr "Windonws:"
```
Comment (stweil) [inline locale/es_ES.UTF-8/LC_MESSAGES/palma.po]: This looks like a typo.

File: locale/es_ES.UTF-8/LC_MESSAGES/palma.po [hunk 2]
```diff
@@ -106,8 +106,8 @@ msgstr "Agregar"
 
 #: index.php:980
 msgid "Sorry! Screensharing for your device is currently not supported."
-msgstr ""
-"Lo lamentamos! Tu dispositivo no está soportado para compartir pantalla."
+msgstr "Lo lamentamos! Tu dispositivo no está soportado para compartir pantalla."
+
```
Comment (stweil) [inline locale/es_ES.UTF-8/LC_MESSAGES/palma.po]: That will be re-formatted automatically to the old format by the `gettext` software which splits "long" texts.

**Comment by stweil** [comment]
The `gettext` software splits longer texts (both for `msgid` and for `msgstr`) into a sequence of strings, so that's normal. The sequence is just the concatenation of the single strings: `"" + "demo"` is the same as `"demo"`.

File: locale/es_ES.UTF-8/LC_MESSAGES/palma.po (hunk 1 above)
Comment (kriegerlukas) [reply locale/es_ES.UTF-8/LC_MESSAGES/palma.po]: @stweil yes, it is fixed by the new commit

**Comment by kriegerlukas** [comment]
> The `gettext` software splits longer texts (both for `msgid` and for `msgstr`) into a sequence of strings, so that's normal. The sequence is just the concatenation of the single strings: `"" + "demo"` is the same as `"demo"`.

ok, so should i remove the change i did on the  ""+<Translation>?

**Comment by stweil** [comment]
Yes, the change for lines 109 / 110 is unnecessary.

**Comment by stweil** [comment]
Thank you, I merged it now.

**Pull request merged**